aspnetcore/includes/RP/model4Win.md

```diff
@@ -7,6 +7,13 @@
   ```console
   dotnet aspnet-codegenerator razorpage -m Movie -dc MovieContext -udl -outDir Pages\Movies --referenceScriptLibraries
   ```
+  
+If you get the error:
+  ```
+No executable found matching command "dotnet-aspnet-codegenerator"
+  ```
+
+Open a command window in the project directory (The directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files).
```

Comment:
"Open a command window in the project directory" and do what? Just open the command window and no further action?

Reply:
You have to read it context.  see https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?branch=pr-en-us-4806 and search for Open a command window

Reply:
Maybe I should make it 
Open **the** command window in the proj dir